src/components/Book.js

```diff
@@ -1,27 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Stats from './Stats';
+import './styles/book.css';
 
 function Book({ book, deleteBook }) {
   return (
-    <div>
-      <li>
-        <div>
-          <p>Title:</p>
-          {book.title}
+    <li className="book-container">
```

Comment:
- [ ] nice job so far... please kindly ensure your design is identical to the zeplin design as much as possible. please the 
- `comment`, `edit` and `remove` should be aligned on the same line like in the original design
- also, kindly ensure the `current chapter,` `add book` and `see progress` text are all aligned in same line please kindly fix that.
- Please kindly ensure that the `add book` button is aligned on the right like the zeplin design. the button is overflowing the border on the right.
your design
![Screenshot (1282)_LI](https://github.com/DevaWinner/book-store/assets/65298048/69fba11f-3b77-47f6-847b-e5a941f6da0c)
![Screenshot (1283)_LI](https://github.com/DevaWinner/book-store/assets/65298048/cabc7235-4d3c-4270-bfbb-66630ee68271)
zeplin design
![Screenshot (1284)](https://github.com/DevaWinner/book-store/assets/65298048/63677e29-53a2-496f-b74d-ef594794c05a)

Reply:
Hello @tufoinnkuo10 ,

Thank you for the review. I must say that this is due to your screen. The design is perfect on my computer screen which is a 1920 x 1080 (16:9) screen resolution.

We weren't required to make it adaptable to different screen sizes hence I can't control how it appears on smaller screens.

Take a look at the screenshot below for the task on my screen.

![Screenshot from 2023-07-29 20-59-54](https://github.com/DevaWinner/book-store/assets/122822848/e8d9436f-0337-4bba-9a01-29449f952cae)


Thank you.